editoast/src/views/train_schedule/mod.rs

```diff
@@ -60,9 +62,15 @@ enum TrainScheduleError {
     #[error("No train ids given")]
     #[editoast_error(status = 400)]
     NoTrainIds,
+    #[error("No train schedules given")]
+    #[editoast_error(status = 400)]
+    NoTrainSchedules,
```

Comment:
What's the difference between `NoTrainSchedules` and `NoTrainIds` ?

Reply:
`NoTrainIds` is used when no ids is provided to retrieve results.
`NoTrainSchedules` is used when no train payload is given to the patch endpoint.